container-host-files/etc/scf/config/role-manifest.yml

```diff
@@ -755,6 +755,7 @@ roles:
   - name: blobstore_url_signer
   - name: route_registrar
   - name: metron_agent
+  tags: [indexed]
```

Comment:
Same as UAA, we should have a comment here on why it's indexed. Also the other tag.

Reply:
I am willing to add the note. Unfortunately I do not know the _why_ myself.
I will ping Vlad.

Reply:
Addressed with commit f6660b9523f40d0308d0af11ae0ac08a8f4decd8